src/main/java/com/spotify/docker/client/messages/EndpointConfig.java

```diff
@@ -90,4 +104,36 @@ public static Builder builder() {
 
     public abstract EndpointConfig build();
   }
+
+  @AutoValue
+  public abstract static class EndpointIpamConfig {
```

Comment:
I do not think it will ever be deserialized. The use case for this object is that users create one and send it to docker, not the other way around. Note that `EndpointIpamConfig` is only used in `EndpointConfig`, which is only used in `NetworkConnection`; the latter two also lack `@JsonCreator`.